html-web-page/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>HTML Web Page</title>
+    <style>
+      table,
+      th,
+      td {
+        border: 1px solid black;
+      }
+
+      td:not(:first-child) {
+        text-align: center;
+      }
+    </style>
+  </head>
+  <body>
+    <div class id="h1"></div>
+    <h1>Jelly of the Month Club</h1>
+    <h2>Make all of your friends jelly of your stock of jelly!</h2>
+    <p>
+      Here at JOTMC Inc., we are constantly striving to supply our customers
+      with the highest quality and most unique jellies on a monthly basis.For
+      the jelly lover in your life (or yourself!), this is the gift that keeps
+      on giving all year round. We offer three tiers of monthly jelly joy, and
+      each one offers their own perks. Here is a breakdowm of each, as well as
+      your monthly price:
+    </p>
+
+    <table>
+      <tr>
+        <th></th>
+        <th>Basic Plan</th>
+        <th>Deluxe Plan</th>
+        <th>Jelly Giant Plan</th>
+      </tr>
+      <tr>
+        <td>Three New curated jellies per month</td>
+        <th>&check;</th>
+        <th>&check;</th>
+        <th>&check;</th>
+      </tr>
+      <tr>
+        <td>Monthly jellies come with commemorative mason jars</td>
+        <th></th>
+        <th>&check;</th>
+        <th>&check;</th>
+      </tr>
+      <tr>
+        <td>Free bonus jars of preserves bi-monthly</td>
+        <th></th>
+        <th>&check;</th>
+        <th>&check;</th>
+      </tr>
+      <tr>
+        <td>Free yearly tickets to Jelly Con</td>
+        <th></th>
+        <th></th>
```

Comment:
`th` should only be used for headers, these empty cells should just be `td` elements.